tests/unittests/graphOptzTest.cpp

```diff
@@ -604,6 +604,46 @@ TEST_F(GraphOptz, ReshapeNoop) {
   EXPECT_TRUE(SN->getType()->dims().equals(shape));
 }
 
+/// Test on the optimation on the case of reshapeNode following splatNode
+TEST_F(GraphOptz, ReshapeAfterSplat) {
```

Comment:
May be add a comment that you try to test a positive and a negative test-cases? In one case the optimization should kick-in because this and that, and in the other case the optimization should not happen because of this and that.

Reply:
Done.